cmake/CheckTrezor.cmake

```diff
@@ -0,0 +1,63 @@
+OPTION(USE_DEVICE_TREZOR "Trezor support compilation" ON)
+OPTION(USE_DEVICE_TREZOR_LIBUSB "Trezor LibUSB compilation" ON)
+OPTION(USE_DEVICE_TREZOR_UDP_RELEASE "Trezor UdpTransport in release mode" OFF)
+
+# Use Trezor master switch
+if (USE_DEVICE_TREZOR)
+    # Protobuf is required to build protobuf messages for Trezor
+    include(FindProtobuf OPTIONAL)
+    find_package(Protobuf)
+    if(NOT Protobuf_FOUND)
+        message(STATUS "Could not find Protobuf")
+    endif()
+
+else()
+    message(STATUS "Trezor support disabled by USE_DEVICE_TREZOR")
+endif()
+
+# Try to build protobuf messages
+if(Protobuf_FOUND AND USE_DEVICE_TREZOR)
```

Comment:
Protobuf can be found but that doesnt mean the python-protobuf module is installed.

Reply:
As I commented in the issue, `python-protobuf` is not required. It is not a dependency, in this PR.

Reply:
Anyway, as I said, if something goes wrong the whole Trezor support is disabled. User is then free to resolve the issue or ignore it.

Reply:
> python-protobuf is not required

Why `if(Protobuf_FOUND` then?

Reply:
`find_package(Protobuf)` has nothing to do with `python-protobuf`.

https://cmake.org/cmake/help/v3.8/module/FindProtobuf.html

"Locate and configure the Google Protocol Buffers library."

Reply:
I am mainly interested in `Protobuf_PROTOC_EXECUTABLE`, `PROTOBUF_INCLUDE_DIRS` - as seen below. Python calls protoc binary, but does not uses anything related to `python-protobuf`

Reply:
Gotcha. OK, so I still recommend removing the python requirement for trezor builds (e.g. couldn't what you are using python for not be done just as easily in cmake alone?).

Reply:
I was thinking about it but python script performs additional modifications to generated message files. Python provides quite nice flexibility in this. I find cmake way for this complex and currently not worth the effort as any default python3 installation will do the job OR python2 with `backports.tempfile` package.

I mean, it is still a soft requirement. It seems to me that python is quite common to be considered a problematic dependency nowadays. Are there users who want to have Trezor support enabled and don't have python installed? Or is it a problem to install? 

Removing python dependency has zero to low priority for me. If you want to remove it give it a try with PR.